.github/workflows/print_github_metrics.py

```
@@ -0,0 +1,306 @@
+            
+        headers = {
+            'Authorization': f'token {token}',
+            'Accept': 'application/vnd.github.v3+json'
+        }
+        
+        try:
+            print(f"🔗 Fetching from GitHub API: {endpoint}")
+            response = requests.get(f"https://api.github.com{endpoint}", headers=headers, timeout=10)
+            response.raise_for_status()
+            data = response.json()
+            print(f"✅ Successfully fetched data from GitHub API")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)